routes/socket/util.js

```diff
@@ -80,9 +84,17 @@ module.exports.sendInProgressGameUpdate = (game, noChats) => {
 				// AEM status is ensured when adding to the subscription list
 				_game.chats = chatWithHidden;
 				_game.chats = combineInProgressChats(_game);
+				if (game.private.commandChats && game.private.commandChats[user]) {
+					_game.chats = _game.chats.concat(game.private.commandChats[user]);
```

Comment:
This feels kinda convoluted, you concat on commandChats, delete _game.chats, and then re-"concat" -- why not just move the concat to after the check? I might be making a mistake but it looks like you have the same identical if statement twice